fix/gsi/anavinfo.rrfs_sd

```diff
@@ -55,11 +56,13 @@ control_vector::
  q        65      1       0.80        -1.0     state    q
  smoke    65      1       1.00        -1.0     state    smoke
  dust     65      1       1.00        -1.0     state    dust
+ coarsepm 65      1       1.00        -1.0     state    coarsepm
 ::
 
 chem_guess::
 !var     level  itracer crtm_use   type    orig_name
  smoke    65      1       12     smoke     smoke
  dust     65      1       12     dust      dust
+ coarsepm 65      1       12     coarsepm  coarsepm
```

Comment:
This is what confused me: coarseepm is added here, but in your GSI PR, you removed src/gsi/setuppm2_5.f90 (line 281). Will corseepm be used or not? Or is this just a placeholder for future?

Reply:
> This is what confused me: coarseepm is added here, but in your GSI PR, you removed src/gsi/setuppm2_5.f90 (line 281). Will corseepm be used or not? Or is this just a placeholder for future?

@guoqing-noaa 
1. An obs operator does not have to use all the variables in control/state
vector
2. Pm10 AOD assimilation will  need it.
3. EnVar is under consideration so the PM da can be combined with dbz DA
with RRFSE ens.